test/webgl/context-state.spec.js

```diff
@@ -51,14 +57,57 @@ test('WebGLState#getParameter (WebGL2)', t => {
   t.end();
 });
 
-test('WebGLState#setParameters', t => {
+test('WebGLState#setParameters with enum and functions keys', t => {
+  const {gl} = fixture;
+
+  resetParameters(gl);
+
+  setParameters(gl, GL_PARAMETER_SETTINGS_ONE_ENUM_FUNCTION);
+
+  for (const key in GL_PARAMETER_SETTINGS_ONE) {
+    const value = getParameter(gl, key);
+    t.deepEqual(value, GL_PARAMETER_SETTINGS_ONE[key],
+      `got expected value ${stringifyTypedArray(value)} for key: ${key}`);
+  }
+  t.end();
+});
+
+test('WebGLState#setParameters function with \'seperate\' versions', t => {
```

Comment:
Clarify a bit for the non-settings expert
" - changing parameters with `***Separate` WebGL setter functions"?